packages/contracts-rfq/contracts/FastBridgeV2.sol

```diff
@@ -44,6 +34,16 @@ contract FastBridgeV2 is Admin, IFastBridgeV2, IFastBridgeV2Errors {
     // @dev the block the contract was deployed at
     uint256 public immutable deployBlock;
 
+    function bridgeStatuses(bytes32 transactionId) public view returns (BridgeStatus status) {
+        return bridgeTxDetails[transactionId].status;
+    }
+
+    function bridgeProofs(bytes32 transactionId) public view returns (uint96 timestamp, address relayer) {
+        uint96 proofBlockTimestamp = bridgeTxDetails[transactionId].proofBlockTimestamp;
+        address proofRelayer = bridgeTxDetails[transactionId].proofRelayer;
+        return (proofBlockTimestamp, proofRelayer);
```

Comment:
When you're using named return variables, you could just assign values to them and skip the explicit return statement - this would be a bit cleaner:
```suggestion
        timestamp = bridgeTxDetails[transactionId].proofBlockTimestamp;
        relayer = bridgeTxDetails[transactionId].proofRelayer;
```

Reply:
much cleaner - thanks